### DISCUSSION TITLE
Welcome to Aqua!

### DISCUSSION THREAD

**Discussion by e6nlaq** [e6nlaq/aqua#23] [General]
# What is this place?

Here you can freely discuss Aqua.

**Comment by fhrk-78** [comment]
質問です。aquaはどうすれば実行できますか？

**Reply by e6nlaq** [reply]
1. [ここ](https://github.com/e6nlaq/aqua/releases)から好きなバージョンの圧縮ファイルをダウンロードし、展開します。
2. `bin/`フォルダ内に、`x64`と`x86`フォルダがあるので、お使いの環境に合わせてどちらかのフォルダを開いてください。

Windowsの場合
3. `x64`または`x86`フォルダを環境変数`PATH`に追加します。

Ubuntuの場合
3. `aqua`を、`~/.local/bin`にコピーした後、`$ chmod 777 aqua`で実行可能にします。

4. ターミナルで`$ aqua`と実行して
```
----------------------------------------------------------------------
Welcome to Aqua For Windows 64bit!

Aqua helps to solve a very small problem
Let's specify the Aqua script as the first argument!
----------------------------------------------------------------------

To exit, press Enter key.
```
と表示されたらインストールは完了です。

5. 好きなところにファイルを作成し、以下のようにしてください。
```
outf "Hello, world!"
```

6. ターミナルをファイルを作成したディレクトリで開き、`$ aqua ファイル名`で実行し、
```
Hello, world!
```

と出力されたら正常に実行できています。

**Reply by fhrk-78** [reply]
出来ました！ありがとうございます！
バッチファイルの様な書式で面白いです！
